frontend/src/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -1336,7 +1335,7 @@ function QueryBuilder(props: QueryBuilderProps) {
 		rules: serializedRules,
 	}: QueryBuilderState = JSON.parse(searchQuery)
 	const startingRules = deserializeRules(serializedRules)
-	const [isAnd, toggleIsAnd] = useToggle(serializedIsAnd)
+	const [isAnd, toggleIsAnd] = useState<boolean>(serializedIsAnd)
```

Comment:
Didn't see a real use case for `useToggle` since we were always passing in the value we want to assign.